app/furniture/marketplace/marketplace.rb

```diff
@@ -10,6 +10,8 @@ class Marketplace < Furniture
 
     has_many :tax_rates, inverse_of: :marketplace
 
+    settings_attribute :notify_emails
```

Comment:
I feel like `setting :notify_emails` is sufficient and shorter; but I feel like `settings_attribute` is already a big win.

Reply:
Renamed!